### ISSUE TITLE
Provide tank relevant statistics

### ISSUE THREAD

**Issue by Bloodmallet** [Bloodmallet/bloodmallet.github.io#181]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by scamille** [comment]
As far as I can tell, to provide relevant tank statistics at bloodmallet.com, the main goal is to increase the dimensions / options you have for each chart, just like you can currently switch between Patchwerk and HecticAddCleave.

I would differentiate between adding dimensions which can be extracted at no extra simulation-time cost from the existing simulations, and those that require extra simulations.

In the first category, I would recommand adding to bloody tools configurable option 'target_metric', which is a list of simc target metrics, and which by default is {dps} for all specs. This option should be configurable at least for each spec, so that you can easily add or remove additional metrics for each spec.
The idea is to eg. set target metrics for brewmaster monk to {dps, tmi}.
In Bloodytools you would only have to add this configuration option, and get the appropriate target metric from the json output.
The good thing is that we do not need to discuss which target metrics we really want just now, but it can maintanbly configured as we go along, depending on the support of the given SimC spec module.

The other dimension to add are the different target tank dummies with their respective damage and armor coefficients. But something like this would add to the simulation space, increasing the combination of simulations to perform.

**Comment by scamille** [comment]
I tried to look at bloody tools a bit:
You could just change https://github.com/Bloodmallet/bloodytools/blob/d8c5f6dbdfdb7d189529c07d2d1f98a084ae116a/bloodytools/simulation_objects/simulation_objects.py#L274 to get_metric_results() and instead of just returning a single dps value, you would return {"dps": X, "tmi": Y, "damagetaken": Z, "hps": U, ...}. You can do that always, since the values are there at no cost, assuming you parse json instead of text report.

Then you will have all the metrics in bloodmallet.github.io/master/json and from there, it would be just a frontend change to show a selection of them for specific specs.

Lookin at the set_dps() function and its callin sites, it looks like you are still parsing the text output. Is that correct?

**Comment by Bloodmallet** [comment]
Yes, you're correct. The switch to use json2 from SimulationCraft as the means to extract information is on my todo for the near future. I'm unsure if I want to try and extract all such information for every spec or only for tank specs. The upside of extracting them all for everybody would obviously be, that everyone could decide on their own which values to look at in each chart.
